src/02-cta/index.js

```diff
@@ -76,8 +76,9 @@ registerBlockType('mojblocks/cta', {
 
         return (
             <div className={`${className}`}>
-                <div className="mojblocks-cta__heading-container">
-                    <h3 className="mojblocks-cta__heading">
+                <div className={'nhsuk-width-container'}>
```

Comment:
are we not removing all nhsuk class styles in our blocks?

Reply:
All done now :)